README.md

```diff
@@ -2,6 +2,10 @@
 
 Make `ssh` login to your Raspberry PI, and go to steps:
 
+Step0:
+
+docker install
```

Comment:
Could you please describe how to install docker-compose or add a ling to instruction?